.github/workflows/publish-docs.yml

```diff
@@ -1,4 +1,4 @@
-name: docs
+name: publish docs
```

Comment:
This shouldn't have spaces.

Reply:
Are you sure? I'm using spaces in a `name:` key value pair with no problems in a workflow file on my own docs repo:

https://github.com/RoostingGeese/git-gosling/blob/main/.github/workflows/docusaurus-build.yml
https://github.com/RoostingGeese/git-gosling/blob/main/README.md

Reply:
The GitHub docs don't give any restriction on the name property, I think.

Quote from https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#name:

> The name of your workflow. GitHub displays the names of your workflows on your repository's actions page. If you omit `name`, GitHub sets it to the workflow file path relative to the root of the repository.

Reply:
Sure, but I don't like the spaces. 😅

Reply:
Ah okay, I'll go add a `-` separator then. 🙃

Reply:
Yes, it's also more url safe. 😁

Reply:
> 
> 
> Yes, it's also more url safe. 😁

I had not considered that yet, that's a good reason to change it then.